documentation/modules/proc-configuring-kafka-bridge-jmx-metrics.adoc

```diff
@@ -1,22 +1,22 @@
-[id='proc-configuring-kafka-bridge-metrics-{context}']
-= Configuring metrics
+[id='proc-configuring-kafka-bridge-jmx-metrics-{context}']
```

Comment:
I think users should follow one method _OR_ the other, otherwise you probably end up with duplicated metrics.

Reply:
Correct. Thanks.